modules/dbnd/src/dbnd/_core/settings/describe.py

```diff
@@ -17,3 +17,6 @@ class DescribeConfig(Config):
     no_tree = parameter(
         default=False, description="Describe without showing tasks tree"
     )[bool]
+    task_visualiser_max_value_size = parameter(
```

Comment:
DescribeConfig is only initalized and used when a command runs with the `--describe` flag.
Everything checks out, this just needs to be moved to the `FeaturesConfig` or to another relevant one :)

Reply:
eh I thought this corresponds to `describe` section in config :P

Reply:
wait, are you sure about that? because I checked it with this test:
```python
        task_visualiser_max_value_size = parameter(
            description="Maximum length of string previewed in TaskVisualiser"
        )[int]
```

and in debugger I can clearly see that `self.task.settings.describe.task_visualiser_max_value_size` is there

Reply:
I am absolutely wrong, resolved :)